src/js/components/PageSection/PageSection.js

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import { Box } from '../Box';
+import { PageContext } from '../Page';
+import { PageSectionPropTypes } from './propTypes';
+
+const PageSection = ({ children, fullBackground, ...rest }) => {
```

Comment:
Nice, I like this component and that the PageContext styling is handled for the user.